outputs_trade.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: index
+title: Trade Outputs
+prev: output_prices.html
+next: outputs_land.html
+gcam-version: v5.3 
+---
+
+## Description of Outputs
+
+Trade is not directly output to the GCAM database. Instead, trade information can be derived by taking the difference between production and consumption of a particular commodity in a particular region. See [quantities](outputs_quantity.html) for information on how to get production and consumption.
```

Comment:
As we changed from HOV to Armingoton for crops and livestock sectors, export and import are direct outputs from our model for those sectors.

Reply:
I've added this to the text. We don't have any trade queries that I see though, so I didn't add a table for it.